src/services/logger.js

```diff
@@ -23,9 +23,11 @@ const CONFIG = {
 
 const TITLE = '[forest] 🌳🌳🌳  ';
 
-module.exports = new (winston.Logger)({
+winston.addColors(CONFIG.colors);
+
+module.exports = winston.createLogger({
   transports: [
-    new (winston.transports.Console)({
+    new winston.transports.Console({
       formatter: (options) => {
```

Comment:
Thanks for your contribution 💪🏼

`formatter` does not look to exist in `winston@3.2.1`.
I'll fix this.